### PR TITLE
Global left nav: update design to flat model

### PR DESCRIPTION
**Pull request by dallashudgens**
## Overview

Updating global left nav component with new flat design.

### Added

n/a

### Removed

Removed sections from nav, no more category switching.
Pruned javascript of unused functions like handling children, section animations & flyouts.

### Changed

Component html updated to match code in Common.
Made accessibility updates for keyboard navigation.
General css updates to fit new design.

## Testing / Reviewing
Made changes to unified-header.html so you can test it

### CODE REVIEW

File: consumables/scss/components/unified-header/_left-nav-trigger.scss [hunk 1]
```diff
@@ -40,178 +55,43 @@
...
-
+  &.bx--left-nav__trigger--active {
     &:hover,
     &:focus {
-      background-color: $color__teal-50;
-
-      span,
+      // span {
+      //   background: transparent;
+      // }
```
Comment (hellobrian) [inline consumables/scss/components/unified-header/_left-nav-trigger.scss]: Commented code. You should be able to delete this.

Comment (dallashudgens) [reply consumables/scss/components/unified-header/_left-nav-trigger.scss]: ah, we must have missed that. We'll get that updated!

File: consumables/scss/components/unified-header/_left-nav-trigger.scss [hunk 2]
```diff
@@ -40,178 +55,43 @@
...
-
-      &:hover,
-      &:focus {
-        background-color: $color__blue-51;
-
-        span:before,
-        span:after {
-          background: $color__white;
-        }
+        background: transparent !important;
```
Comment (hellobrian) [inline consumables/scss/components/unified-header/_left-nav-trigger.scss]: You should be okay to remove `!important`

File: consumables/scss/components/unified-header/_left-nav-trigger.scss [hunk 3]
```diff
@@ -11,6 +11,20 @@
   cursor: pointer;
   z-index: 9999;
 
+  &:focus {
+    outline: -webkit-focus-ring-color auto 5px;
```
Comment (hellobrian) [inline consumables/scss/components/unified-header/_left-nav-trigger.scss]: For consistency, can you use the focus-outline mixin? 

```scss
@import '../../global/helpers';

.bx--left-nav__trigger {
  ...

  &:focus {
    // outline: -webkit-focus-ring-color auto 5px;
    @include focus-outline;
  }
  ...
}
```

This just helps us to know where we're applying focus styles.

**Review by hellobrian — APPROVED** [review]
👍 ✅ ❗️ Just need to address some comments but otherwise, everything looks great

**Comment by dallashudgens** [comment]
@hellobrian - @aliciahickman addressed your comments (for some reason she can't make comments on this PR?)

**Comment by chrisdhanaraj** [comment]
Not... sure why @aliciahickman can't, what's the error? With the removal of prior functionality, how would you handle an A/B test? I know that was floated as a possible requirement

**Comment by aliciahickman** [comment]
@chrisdhanaraj I was getting 'you can't perform this action at this time' notification... but i guess it's ok today /shrug

**Comment by aliciahickman** [comment]
...also we checked with Mark and A/B testing is not happening.

**Comment by chrisdhanaraj** [comment]
:+1: no problems from me then, merging it in

**Pull request merged**